### PR TITLE
Adds a notification service

### PR DESCRIPTION
**Pull request by FlorianVeaux**
Also cleans and update a few angular packages

### CODE REVIEW

File: frontend/src/app/home/home.component.ts [hunk 1]
```diff
@@ -14,14 +15,12 @@ export class HomeComponent implements OnInit {
     is_settings_displayed: boolean;
     widgets: {"name": string, "displayed": boolean}[]
     associations: any[];
-    status_message_html: string;
 
-    constructor(private _apiService: ApiService) {
+    constructor(private _apiService: ApiService, private notifier: NotifierService) {
```
Comment (14alarro) [inline frontend/src/app/home/home.component.ts]: _apiService mais notifier : pourquoi pas la même convention (sur l'underscore comme sur le suffixe "Service") ?

Comment (FlorianVeaux) [reply frontend/src/app/home/home.component.ts]: The leading underscore looks like bad practice (https://angular.io/guide/styleguide#style-03-04), we should follow the same convention everywhere though

Comment (14alarro) [reply frontend/src/app/home/home.component.ts]: Agreed, I would write apiService and notifierService

Comment (FlorianVeaux) [reply frontend/src/app/home/home.component.ts]: 👍

**Pull request merged**